app/src/main/java/ca/unb/mobiledev/reflexrevolution/instructions/SwipeInstruction.java

```diff
@@ -12,6 +12,13 @@
     private final TouchDetector touchDetector;
     private TouchDetector.SwipeAction currentAction;
     private Random rand;
+    private Random rand2;
```

Comment:
We don't need a separate random number generator, we can use the same one (it's random after all, so it doesn't change anything).